src/osm_configurator/model/project/configuration/attribute_enum.py

```diff
@@ -66,3 +66,13 @@ def calculate_attribute_value(self, category: Category, tags: Dict[str, str]) ->
             float: the calculated value
         """
         return self.value[2](category, tags)
+
+    @classmethod
+    def get_all_tags(cls) -> List:
```

Comment:
obsolete, i have rewritten this method, either use mine or be careful that it doesnt overwrite mine in the merge.